vm/src/tests/cairo_pie_test.rs

```diff
@@ -6,12 +6,6 @@ use crate::{
 #[cfg(target_arch = "wasm32")]
 use wasm_bindgen_test::*;
 
-#[cfg(all(not(feature = "std"), feature = "alloc"))]
```

Comment:
same here, these imports where unused?

Reply:
Correct. The weren't being used and they also were duplicated.